tools/protodoc/protodoc.py

```diff
@@ -61,6 +61,19 @@
 
   $security_posture
 
+  This extension extends the following extension category:
+
+$categories
```

Comment:
As discussed offline, suggest moving outside the note.

Reply:
ive shifted this into a `tip` also - partly because the equivalence with the `tip` in the categories is helpful in terms of navigation - but also because i think it needs some kind of highlighting of "this is where this can go"